pkg/virt-handler/selinux/labels.go

```diff
@@ -10,13 +10,26 @@ import (
 type SELinuxImpl struct {
 }
 
-func NewSELinux() (SELinux, error) {
-	return &SELinuxImpl{}, isPresent()
+func NewSELinux() (SELinux, bool, error) {
+	present, err := isPresent()
+	return &SELinuxImpl{}, present, err
 }
 
-func isPresent() (err error) {
-	_, err = os.Stat("/proc/1/root/usr/bin/chcon")
-	return
+func isPresent() (present bool, err error) {
+	_, err = os.Stat("/proc/1/root/usr/sbin/getenforce")
+	if os.IsNotExist(err) {
+		return false, nil
+	} else if err != nil {
+		return false, err
+	}
+	out, err := exec.Command("/usr/bin/chroot", "--mount", "/proc/1/ns/mnt", "exec", "--", "/usr/sbin/getenforce").CombinedOutput()
```

Comment:
Do you even need to check to see if SELinux is enabled or not? if the setenforce binary exists, doesn't that mean SELinux is at least installed. Shouldn't the same logic be applied regardlesss if selinux is set to enforcing or disabled at the moment this check occurs?

Reply:
Yes, because I can't do the changes if it is Disabled, only when it is Envorced or Permissive. It would be hard to see if a command should work and fails on an arbitrary error, or libvirt is disabled.